static/app/views/alerts/rules/metric/ruleConditionsForm.tsx

```diff
@@ -141,6 +218,36 @@ class RuleConditionsForm extends PureComponent<Props, State> {
     }
   }
 
+  getEventFieldValues = async (tag, query): Promise<string[]> => {
```

Comment:
This is essentially a rewrite of the `searchBar.tsx` method `getEventFieldValues`